content/docs/get-started/aws/deploy-changes.md

```diff
@@ -12,57 +12,369 @@ menu:
+First Pulumi will run the `preview` step of the update. In the future you can run `pulumi up -y` to automatically run the update after a successful preview.
 
 ```
 Previewing update (dev):
 
-     Type                 Name            Plan       Info
-     pulumi:pulumi:Stack  quickstart-dev
- +   ├─ aws:kms:Key       my-key          create
- ~   └─ aws:s3:Bucket     my-bucket       update     [diff: +serverSideEncryptionConfiguration]
+     Type                    Name            Plan       Info
+     pulumi:pulumi:Stack     quickstart-dev
+ +   └─ aws:s3:BucketObject  index.html                 create
 
 Resources:
     + 1 to create
-    ~ 1 to update
-    2 changes. 1 unchanged
+    2 unchanged
 
 Do you want to perform this update?
-  yes
-> no
+> yes
+  no
   details
 ```
 
-Pulumi will create the KMS key and update the bucket with the new encryption configuration.
-
-Choosing `yes` will proceed with the update.
+Choosing `yes` will proceed with the update and upload your `index.html` file to your bucket.
 
 ```
 Do you want to perform this update? yes
 Updating (dev):
 
-     Type                 Name            Status      Info
-     pulumi:pulumi:Stack  quickstart-dev
- +   ├─ aws:kms:Key       my-key          created
- ~   └─ aws:s3:Bucket     my-bucket       updated     [diff: +serverSideEncryptionConfiguration]
+     Type                     Name            Status      Info
+     pulumi:pulumi:Stack      quickstart-dev
+  +   └─ aws:s3:BucketObject  index.html                  created
 
 Outputs:
-    bucket_name: "my-bucket-de014a8"
+    bucketName: "my-bucket-68e33ec"
 
 Resources:
     + 1 created
-    ~ 1 updated
-    2 changes. 1 unchanged
+    2 unchanged
+
+Duration: 6s
+```
+
+Once the update has completed, rerun the command to list the contents of your bucket.
+
+{{< chooser language "javascript,typescript,python,go,csharp" / >}}
+
+{{% choosable language javascript %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output bucketName)
+```
+
+{{% /choosable %}}
+
+{{% choosable language typescript %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output bucketName)
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output bucket_name)
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output bucketName)
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output BucketName)
+```
+
+{{% /choosable %}}
+
+Notice that your index.html file has been added:
+
+```bash
+2020-08-27 12:30:24         68 index.html
+```
+
+{{% choosable language javascript %}}
+Now that your `index.html` is in your bucket, let's modify the program to have the bucket serve the file as a static website. First you need to set the `website` property on your bucket.
+
+```javascript
+const bucket = new aws.s3.Bucket("my-bucket", {
```

Comment:
I followed the steps in the workshop: https://pulumi.awsworkshop.io/20_getting_started_with_pulumi.html

But doing it this way does get the user to do 3 updates instead of 2 and I would love to see if there is an engagement difference between 2 and 3 updates.